app/ModelFunctions/PhotoFunctions.php

```diff
@@ -140,6 +140,54 @@ public function createThumb(Photo $photo, string $frame_tmp = '')
 		return true;
 	}
 
+	/**
+	 * Returns 'photo' if it is a photo
+	 * Returns 'video' if it is a video
+	 * Returns 'raw' if it is an accepted file (we only check extensions)
+	 * Returns 'error message' if it is something else.
+	 *
+	 * @param $file
+	 * @param $extension
+	 *
+	 * @return string
+	 */
+	private function kind($file, string $extension)
+	{
+		// check raw files
+		if (in_array(strtolower($extension), explode('|', Configs::get_value('raw_formats', '')), true)) {
+			return 'RAW';
```

Comment:
I don't really like the use of uppercase `'RAW'` here but lowercase `'raw'` everywhere else. Any reason for this?

Reply:
not really. Just that I prefer constants that are used for decision to be in uppercase. I don't really care to be honest.

Reply:
I think consistency is important, especially in the same function's return values